PS4k/src/com/vitco/layout/content/menu/MainMenuLogic.java

```diff
@@ -532,6 +534,59 @@ protected Object doInBackground() throws Exception {
 
         // ---------------
 
+        // add "turntable image sequence" export
+        FieldSet turnTableImageSeq = new FieldSet("turntable_image_sequence", "Turntable Render (*.png)");
+        turnTableImageSeq.addComponent(new LabelModule("Render as multiple image sequence"));
+        final TextInputModule frameCount = new TextInputModule("frames_count", "Number of frames", "60", true);
+        frameCount.setVisibleLookup("export_type=turntable_image_sequence");
+
+        turnTableImageSeq.addComponent(frameCount);
+        turnTableImageSeq.addComponent(new CheckBoxModule("watermark", "Add watermark: Show VoxelShop some love", true));
+        turnTableImageSeq.addComponent(new SeparatorModule("Animation orbit:"));
+        turnTableImageSeq.addComponent(new ComboBoxModule("animation_orbit", new String[][]{
+                new String[]{"clockwise", "ClockWise"},
+                new String[]{"anti-clockwise", "Anti-Colockwise"},
+        }, 0));
+        turnTableImageSeq.addComponent(new SeparatorModule(""));
+        turnTableImageSeq.addComponent(new CheckBoxModule("add_background_color", "Add Image background color", false));
+        TextInputModule TISBackgroundColor = new TextInputModule("background_color", "Image Background Color: e.g #ffffff", "#000000", true);
+        TISBackgroundColor.setVisibleLookup("export_type=turntable_image_sequence&turntable_image_sequence.add_background_color=true");
+        TISBackgroundColor.setEnabledLookup("export_type=turntable_image_sequence&turntable_image_sequence.add_background_color=true");
+        turnTableImageSeq.addComponent(TISBackgroundColor);
+
+        // ---------------
+
+        // add "turntable animation" export
+        FieldSet turntableAnim = new FieldSet("turntable_animation", "Turntable Render (*.gif)");
```

Comment:
The two does the same thing in different ways. The animation (.gif) is mostly useful if you want to quickly showcase your model on social media but the image sequence will be helpful to those who want to use the render in third party application e.g video/image editor. Merging the two together will either create a mess, give user less options or just make the list longer. BTW here on linux i experience issues of not been able to scroll in the export dialog (colladae). Another option is to remove it from export and include it in the toolbar like how QBCL does it.